wide_deep/torch_model.py

```diff
@@ -107,7 +106,7 @@ def compile(self, method="logistic", optimizer="Adam", learning_rate=0.001, mome
         if method == 'regression':
             self.activation, self.criterion = None, F.mse_loss
         if method == 'logistic':
-            self.activation, self.criterion = F.sigmoid, F.binary_cross_entropy
+            self.activation, self.criterion = torch.sigmoid, F.binary_cross_entropy
```

Comment:
why `torch.sigmoid`? the functional `F.sigmoid` is actually more efficient?

Reply:
@jrzaurin 
- You're right.  I heard the functional backward code `F.sigmoid` was efficient in versions below 0.3.x.
- But in all later versions (>0.4.x, stable, etc.), `nn.functional.sigmoid` is deprecated.
https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1379
So I suggest using `torch.sigmoid` instead. :)